hexa/databases/utils.py

```diff
@@ -0,0 +1,109 @@
+from typing import Dict, List, Tuple
+
+import psycopg2
+from psycopg2 import sql
+
+from hexa.core.graphql import result_page
+from hexa.workspaces.models import Workspace
+
+
+def format_sample_data(columns, samples):
+    res = []
+    for sample in samples:
+        res.append(
+            [{"name": columns[j], "value": sample[j]} for j in range(len(sample))]
+        )
+    return res
+
+
+def get_database_definition(
+    workspace: Workspace, limit_per_table=4, page=1, per_page=10
```

Comment:
You should not get the sample data with this function. It takes too much time.

Reply:
You suggest to create a resolver for the sample field.

Reply:
Yup